### PR TITLE
Check console enabled log in /log/messages

### PR DESCRIPTION
**Pull request by adityagesh**
console enabled log may get flushed from dmesg.
Hence check for pattern in /var/log/messages.

Change assert statement for better error readability

### CODE REVIEW

File: microsoft/testsuites/core/azure_image_standard.py [hunk 1]
```diff
@@ -745,16 +745,27 @@ def verify_serial_console_is_enabled(self, node: Node) -> None:
             CpuArchitecture.X64: "ttyS0",
             CpuArchitecture.ARM64: "ttyAMA0",
         }
+        messages_log_file = "/var/log/messages"
```
Comment (LiliDeng) [inline microsoft/testsuites/core/azure_image_standard.py]: for centos/rhel, it is /var/log/syslog

Comment (adityagesh) [reply microsoft/testsuites/core/azure_image_standard.py]: updated

File: microsoft/testsuites/core/azure_image_standard.py [hunk 2]
```diff
@@ -745,16 +745,30 @@ def verify_serial_console_is_enabled(self, node: Node) -> None:
...
         arch = lscpu.get_architecture()
         current_console_device = console_device[CpuArchitecture(arch)]
-        console_enabled_string = f"console [{current_console_device}] enabled"
-        dmesg = node.tools[Dmesg]
+        console_enabled_pattern = re.compile(
+            rf"^(.*console \[{current_console_device}\] enabled.*)$", re.M
+        )
+
+        cat = node.tools[Cat]
+        if node.shell.exists(node.get_pure_path("/var/log/messages")):
```
Comment (LiliDeng) [inline microsoft/testsuites/core/azure_image_standard.py:758]: shell.exists can't check file which need sudo permission.

File: microsoft/testsuites/core/azure_image_standard.py [hunk 3]
```diff
@@ -732,7 +732,7 @@ def verify_repository_installed(self, node: Node) -> None:  # noqa: C901
          in dmesg.
 
         Steps:
-        1. Get the kernel command line from dmesg output.
+        1. Get the kernel command line from /var/log/messages output.
```
Comment (LiliDeng) [inline microsoft/testsuites/core/azure_image_standard.py]: also need update this

Comment (adityagesh) [reply microsoft/testsuites/core/azure_image_standard.py]: updated

**Pull request merged**